pb/activeconnection_proto/activeconnection.proto

```diff
@@ -26,4 +26,7 @@ message ActiveConn{
     int32 UID = 4;
     int32 SensorNumber = 5;
     int32 PendingFlush = 6;
+    int32 SamplingPeriod = 7;
```

Comment:
Add units to name. It should be SamplingPeriodNs or SamplingPeriodMs. Same is true for Batching Period.